app/services/patient_action_or_outcome_service.rb

```diff
@@ -0,0 +1,25 @@
+class PatientActionOrOutcomeService
+  def self.call(consent:, triage:, vaccination_record:)
+    if consent.nil?
+      { action: :get_consent }
+    elsif consent&.consent_refused?
+      { action: :check_refusal }
+    elsif consent&.consent_given? && consent&.triage_needed? && triage.blank? &&
+          vaccination_record.nil?
+      { action: :triage }
+    elsif consent&.consent_given? && !consent.triage_needed? &&
+          vaccination_record.nil?
+      { action: :vaccinate }
+    elsif triage&.status == "ready_for_session" && vaccination_record.nil?
```

Comment:
Pretty sure you can check status on `triage` the same way you have on `consent`:

```suggestion
    elsif triage&.ready_for_session? && vaccination_record.nil?
```

Reply:
fixed in [daa4d2e](https://github.com/nhsuk/record-childrens-vaccinations/pull/280/commits/daa4d2ece6719084cceae5fc2263a8b9ea0923ec)